src/mono/wasm/data/aot-tests/Directory.Build.props

```
@@ -13,16 +13,20 @@
     <WASI_SDK_PATH>$(WasmBuildSupportDir)\wasi-sdk\</WASI_SDK_PATH>
   </PropertyGroup>
 
-  <PropertyGroup >
-    <_WasmTargetsDir Condition="'$(_WasmTargetsDir)' == '' and '$(RuntimeSrcDir)' != '' and '$(RuntimeIdentifier)' == 'browser-wasm'">$(RuntimeSrcDir)\src\mono\wasm\build\</_WasmTargetsDir>
+  <PropertyGroup Condition="'$(RuntimeSrcDir)' != ''">
+    <_WasmTargetsDir Condition="'$(_WasmTargetsDir)' == '' and '$(RuntimeSrcDir)' != '' and '$(RuntimeIdentifier)' == 'browser-wasm'">$(RuntimeSrcDir)\src\mono\browser\build\</_WasmTargetsDir>
```

Can we use `BrowserProjectRoot` here?

```suggestion
    <_WasmTargetsDir Condition="'$(_WasmTargetsDir)' == '' and '$(RuntimeSrcDir)' != '' and '$(RuntimeIdentifier)' == 'browser-wasm'">$(BrowserProjectRoot)\build\</_WasmTargetsDir>
```

---

This gets used outside the repo, on helix to be specific, so the property won't be set there.